config/locales/en.yml

```diff
@@ -81,6 +81,8 @@ en:
         update_sample?: You are not authorized to update samples for project %{name} on this server.
         transfer_sample?: You are not authorized to transfer samples for project %{name} on this server.
         transfer_sample_into_project?: You are not authorized to transfer samples to project %{name} on this server.
+        clone_sample?: You are not authorized to clone samples for project %{name} on this server.
```

Comment:
Update the `for` to `from` in this translation

Reply:
0511ed1811d36d87b3023ab99fa640ae5cb88edf